gusto_embedded/README.md

```diff
@@ -94,29 +94,92 @@ yarn add @tanstack/react-query react react-dom
 For supported JavaScript runtimes, please consult [RUNTIMES.md](RUNTIMES.md).
 <!-- End Requirements [requirements] -->
 
-<!-- Start SDK Example Usage [usage] -->
 ## SDK Example Usage
 
 ### Example
+Automatic token refresh using a persistent data store.
 
 ```typescript
 import { GustoEmbedded } from "@gusto/embedded-api";
+import { CompanyAuthenticatedClient } = "@gusto/embedded-api/CompanyAuthenticatedClient";
 
-const gustoEmbedded = new GustoEmbedded({
-  companyAccessAuth: process.env["GUSTOEMBEDDED_COMPANY_ACCESS_AUTH"] ?? "",
-});
+class PersistentTokenStore implements TokenStore {
```

Comment:
Do we need to add some kind of documentation that this is insecure if you attempt to use it in the browser? Right now the SDK assumes it's running solely in the browser and not on the server. That's subject to change eventually but it might break some assumptions we currently make.

Reply:
The reason I bring up the SDK here is that it ships this library, so whatever imports we add here have a chance of intentionally or unintentionally impacting the SDK